src/components/ingredientCard/IngredientCard.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useMemo } from 'react';
+import { View, Animated, useAnimatedValue } from 'react-native';
+import FastImage from 'react-native-fast-image';
+import LinearGradient from 'react-native-linear-gradient';
+
+import TextBlock from '@components/customText';
+import useThemeContext from '@hooks/useThemeContext';
+import useStyles from '@hooks/useStyles';
+
+import ingredientCardStyles from './styles';
+
+function IngredientCard({ imageUrl, name, amount, unit }: IngredientCardProps) {
+  const position = useAnimatedValue(100);
+
+  const { gradients } = useThemeContext();
+
+  const styles = useStyles(ingredientCardStyles);
+
+  const animatedStyles = useMemo(() => {
```

Comment:
can make this a wrapper component as `FadeInSlideUp`

Reply:
I removed the animation from `IngredientCard` component as I am directly animating the whole ingredient card list with `FadeInSlideUp` and also added the slide In animation component for later usage.